engine/runtime/src/main/java/org/enso/interpreter/service/error/ExitException.java

```diff
@@ -0,0 +1,63 @@
+package org.enso.interpreter.service.error;
+
+import com.oracle.truffle.api.TruffleStackTraceElement;
+import com.oracle.truffle.api.exception.AbstractTruffleException;
+import com.oracle.truffle.api.interop.ExceptionType;
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.library.ExportLibrary;
+import com.oracle.truffle.api.library.ExportMessage;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeHelpers;
+
+/** Thrown when {@code System.exit} call was called during execution. */
+@ExportLibrary(InteropLibrary.class)
+public class ExitException extends AbstractTruffleException implements ServiceException {
```

Comment:
`final`

Reply:
Marked as `final` in 76a2b864fd44076256fd5c5185a31a96c52d179a